docs/blockchain_tests/index.rst

```diff
@@ -0,0 +1,139 @@
+.. _blockchain_tests:
+
+################################################################################
+Blockchain Tests
+################################################################################
+
+Found in ``/BlockTests``, the blockchain tests aim is to test the basic verification of a blockchain.
+
+``/BlockTests`` - general blockchain tests. All blocks are built on network: **Frontier** 
+``/BlockTests/Homestead`` - homestead blockchain tests. All blocks are built on network: **Homestead** 
```

Comment:
this info is outdated. blockchain folder structure is different now. network rules are read from networ field from within the test